ietf-te-types.yang

```diff
@@ -2237,61 +2245,57 @@ module ietf-te-types {
           "Traffic Engineering (TE) Link Metric.";
         reference
           "RFC 3630: Traffic Engineering (TE) Extensions to OSPF
-          Version 2, section 2.5.5
-
-          RFC 5305: IS-IS Extensions for Traffic Engineering,
-          section 3.7";
+                     Version 2, Section 2.5.5
+           RFC 5305: IS-IS Extensions for Traffic Engineering,
+                    Section 3.7";
       }
 
-      identity link-metric-igp {
+      identity igp {
```

Comment:
Same comment as per admin-up

Reply:
There is no link-metric-igp in 8776. 

You don't need to repeat the name of the identity as this will be prefixed anayway.

Reply:
My understanding is that identities are prefixed with the module name and not with the base identity

No strong opinion for me (as opposed to the admin state, we have no other identity called 'igp in the module) but I noted that the authors of RFC 8776 have adopted a different convention

Let's discuss also with them

Reply:
Within this module we have:
- link-metric-igp
- path-metric-igp
- svec-metric-cumulative-igp

A prefix is therefore needed to differentiate these three identities.

```suggestion
      identity link-metric-igp {
```